features/product/viewing_products/redirecting_to_products_when_there_is_a_trailing_slash_in_path.feature

```diff
@@ -0,0 +1,21 @@
+@viewing_products
+Feature: Redirecting to products when there is a trailing slash in path
+    In order to be redirected on products list
+    As a Visitor
+    I want to be redirected on products list when path has a trailing slash
+
+    Background:
+        Given the store has currency "Euro"
+        And the store operates on a channel named "Poland"
+        And the store classifies its products as "T-Shirts", "Funny" and "Sad"
+        And the store has a product "T-Shirt Banana" available in "Poland" channel
+        And this product belongs to "T-Shirts"
+        And the store has a product "Plastic Tomato" available in "Poland" channel
+        And this product belongs to "Funny"
+
+    @ui
```

Comment:
```suggestion
    @ui @no-api
```

but TBH, I would prefer to do just simple PHPUnit test for that instead of behat